packages/vscode-extension/src/handlers.ts

```diff
@@ -624,6 +624,83 @@ export async function addCapabilityHandler(args?: any[]): Promise<Result<null, F
   return result;
 }
 
+export async function addFeatureHandler(args?: any[]): Promise<Result<null, FxError>> {
+  // TODO: move the question model to fx-core addFeature API
```

Comment:
Send telemetry here?

Reply:
This function delegates call to other existing handlers and they have their own telemetry events. I'd like to add telemetry for "Add feature" once the API in core is ready.